[ES] Add `Legal/Privacy` (#11986)
ppy/osu-wiki: wiki/Legal/Privacy/es.md

---
legal: true
---

# Política de privacidad de osu!

> Actualizado por última vez el 16 de abril de 2021. [Véase el historial de cambios aquí](https://github.com/ppy/osu-wiki/commits/master/wiki/Legal/Privacy/en.md)

Reviewer: ```suggestion
Actualizado por última vez el 16 de abril de 2021. [Consulte el historial de cambios aquí](https://github.com/ppy/osu-wiki/commits/master/wiki/Legal/Privacy/en.md)
```

Reviewer: me parece más bonito

> Actualizado por última vez el 16 de abril de 2021. [Véase el historial de cambios aquí](https://github.com/ppy/osu-wiki/commits/master/wiki/Legal/Privacy/en.md)

Reviewer: ```suggestion
Actualizado por última vez el 16 de abril de 2021. [Consulte el historial de cambios aquí](https://github.com/ppy/osu-wiki/commits/master/wiki/Legal/Privacy/en.md)
```

> Actualizado por última vez el 16 de abril de 2021. [Comprueba el historial de cambios aquí](https://github.com/ppy/osu-wiki/commits/master/wiki/Legal/Privacy/en.md)

Reviewer 2: ```suggestion
Actualizado por última vez el 16 de abril de 2021. [Véase el historial de cambios aquí](https://github.com/ppy/osu-wiki/commits/master/wiki/Legal/Privacy/en.md)
```

En adición a esta política, por favor asegúrese también de visitar y entender nuestros [Términos de servicio](/wiki/Legal/Terms).

## Introducción

osu! es dirigido por una entidad australiana (ppy Pty Ltd), con la mayoría de los servidores operando desde los Estados Unidos de América.

osu! ofrece una amplia variedad de servicios a usuarios de casi todos los países del mundo, con una comunidad animada y muchas oportunidades para los usuarios de compartir su creatividad con otros mediante perfiles de usuario, beatmaps (niveles del juego), foros, mensajes privados, chats dentro del juego, comentarios en las repeticiones.

Para poder ofrecerle estos servicios, debemos habitualmente recoger, almacenar y transferir información personal. Nos esforzamos por ser lo más transparentes posible en todos los aspectos relacionados con la gestión de osu!, y esta política de privacidad nos debería traer aún más cerca a ese objetivo.

> Este documento busca describir que información recogemos en nuestra red y, en el uso de nuestros productos y servicios, cómo usamos esa información y que opciones le ofrecemos para controlar su información personal.

Reviewer 2: ```suggestion
Este documento busca describir que información recogemos en nuestra red y, en el uso de nuestros productos y servicios, cómo usamos esa información y qué opciones le ofrecemos para controlar su información personal.
```

> Al usar nuestro servicio, confirma que ha leido y comprendido los [Terminos de servicio](/wiki/Legal/Terms) y esta política de privacidad, incluyendo cómo y porqué usamos su información y que su uso del servicio está sujeto a los Terminos de Servicio aplicables y esta Política de Privacidad. Si deseas que no recojamos o procesemos su información personal como se describen en este documento, tiene distintas opciones incluyendo la limitación de información que recogemos de Ud., o no usar la red, o nuestros Productos y Servicios.

Author: "Productos y Servicios" está capitalizado así en el inglés asi que lo dejo así por el momento

```suggestion
Al usar nuestro servicio, confirma que ha leído y comprendido los [Términos de servicio](/wiki/Legal/Terms) y esta política de privacidad, incluyendo cómo y porqué usamos su información y que su uso del servicio está sujeto a los Términos de Servicio aplicables y esta Política de Privacidad. Si deseas que no recojamos o procesemos su información personal como se describe en este documento, tiene distintas opciones, como limitar la información que recogemos de usted, o no usar la red, o nuestros Productos y Servicios.
```

> Al usar nuestro servicio, confirma que ha leído y comprendido los [Términos de servicio](/wiki/Legal/Terms) y esta política de privacidad, incluyendo cómo y por qué usamos su información y que su uso del servicio está sujeto a los Términos de Servicio aplicables y esta Política de Privacidad. Si desea que no recojamos o procesemos su información personal como se describe en este documento, tiene distintas opciones, como limitar la información que recogemos de usted, o no usar la red, o nuestros Productos y Servicios.

Reviewer: ```suggestion
Al usar nuestro servicio, confirma que ha leído y comprendido los [términos de servicio](/wiki/Legal/Terms) y esta política de privacidad, incluyendo cómo y por qué usamos su información y que su uso del servicio está sujeto a los términos de servicio aplicables y esta política de privacidad. Si desea que no recojamos o procesemos su información personal como se describe en este documento, tiene distintas opciones, como limitar la información que recogemos de usted, o no usar la red, o nuestros productos y servicios.
```

Reviewer: ahora

Author: vaaale

> Al usar nuestro servicio, confirma que ha leído y comprendido los [Términos de servicio](/wiki/Legal/Terms) y esta política de privacidad, incluyendo cómo y porqué usamos su información y que su uso del servicio está sujeto a los Términos de Servicio aplicables y esta Política de Privacidad. Si deseas que no recojamos o procesemos su información personal como se describe en este documento, tiene distintas opciones, como limitar la información que recogemos de usted, o no usar la red, o nuestros Productos y Servicios.

Reviewer 2: ```suggestion
Al usar nuestro servicio, confirma que ha leído y comprendido los [Términos de servicio](/wiki/Legal/Terms) y esta política de privacidad, incluyendo cómo y por qué usamos su información y que su uso del servicio está sujeto a los Términos de Servicio aplicables y esta Política de Privacidad. Si deseas que no recojamos o procesemos su información personal como se describe en este documento, tiene distintas opciones, como limitar la información que recogemos de usted, o no usar la red, o nuestros Productos y Servicios.
```

## Información que recogemos

### Al registrar una cuenta

> Mientras que una limitada parte de la funcionalidad se puede disfrutar sin una cuenta, es normalmente necesario que un usuario registre una cuenta para experimentar ciertos servicios. Cuando registra una cuenta, almacenamos

Reviewer 2: ```suggestion
Si bien es posible acceder a ciertas funciones sin tener una cuenta, a menudo es necesario que el usuario registre una cuenta poder acceder a ciertos servicios. Al registrar una cuenta, almacenamos
```

> - Su nombre de usuario

Reviewer: puedes usar "tu" igualmente

Author: terminos y condiciones/politicas de privacidad etc etc se suelen dejar todo lo 3a persona que sea posible

Author: porque esto es algo serio, no está dirigido a los jugadores en general sino a los que están preocupados por sus datos, y abogados, juzgados, etc. 

En un texto legal hay que mantener la formalidad, y en español eso conlleva hablar de usted.

(Por cierto, sería segunda persona, no primera)

Reviewer 2: Bueno, no le voy a dar mucha vuelta entonces

Reviewer: a quien le importa, arriba de la página pone que la única versión aplicable es la que está en inglés, además ya puse primera persona en el del derecho musical 😝

Reviewer: .

Author: prefiero poner el derecho musical en 3a persona, aunque carezca de validez legal sigue manteniendo las formas y eso puede atraer/ayudar/whatever a artistas que sean de españa e hispanoamérica que no quieran leer la parte en inglés, lo hace más oficial y respetable imo

- Su dirección de correo electrónico
- Su contraseña (bcrypt+salt)
- Su dirección IP y su país

Esta información personal, con la excepción de su nombre de usuario y país, nunca se comparte públicamente.

### Al actualizar su perfil

> Cuando complete su perfil de usuario (el cual es publicamente visible para el resto de usuarios), puede proporcionar opcionalmente

Reviewer 2: ```suggestion
Al crear su perfil de usuario (que es visible públicamente para todos los demás usuarios), puede proporcionar opcionalmente
```

- Su ubicación actual
- Sus intereses
- Su ocupación
- Su presencia en redes sociales (twitter, discord, skype, una página web)

> - La imagen de su avatar y el encabezado de su perfil

Reviewer: ```suggestion
- La imagen de su avatar y la portada de su perfil
```

- Su firma

Todos los campos anteriores son visibles públicamente, pero pueden retirarse de forma inmediata y permanente en cualquier momento desde [la página de configuración](https://osu.ppy.sh/home/account/edit).

### Al subir contenido enviado por el usuario

> Al publicar en los foros, participar en el chat o subir contenido a nuestro servicio, como puede ser un beatmap, estás expresamente publicando todo el contenido subido. En la mayoría de los casos, se puede editar y eliminar después de subirlo bajo su propia discreción, pero en algunas ocasiones, esta funcionalidad puede estar desactivada para mantener la relevancia y la prestación del servicio a nuestra base de usuarios.

Reviewer 2: ```suggestion
Al publicar en los foros, participar en el chat o subir contenido a nuestro servicio, como un beatmap, usted está haciendo público expresamente todo el contenido subido. En la mayoría de los casos, puede editarse y borrarse a su discreción, pero en algunas ocasiones, esta función puede desactivarse para mantener la relevancia y la prestación del servicio a nuestra base de usuarios.
```

> Por ejemplo, si sube un beatmap y está «clasificado», este se convierte en el fundamento para la base de usuarios en general en el que conseguir puntuaciones. Llegados a este punto, la posibilidad de eliminar el contenido será revocada.

Author: en el original se usa ranked en mencion al estado del mapa, no como verbo pero es opinión personal, puede quedar mejor pero no lo incluyo por ahora

> Por ejemplo, si sube un beatmap y está «clasificado», este se convierte en el fundamento para la base de usuarios en general en el que conseguir puntuaciones. Llegados a este punto, la posibilidad de eliminar el contenido será revocada.

Reviewer 2: No puedes subir un mapa y que ya esté clasificado, este se tiene que 'clasificar'

> Por ejemplo, si sube un beatmap y está «clasificado», este se convierte en el fundamento para la base de usuarios en general en el que conseguir puntuaciones. Llegados a este punto, la posibilidad de eliminar el contenido será revocada.

Author: fair

### Al iniciar sesión en el cliente del juego

> Cuando se conecta a nuestro servicio desde el cliente de juego de osu!, una cadena específica del cliente es subida para ayudarnos a identificar su entorno de juego actual. Se crea a partir de una combinación de identificadores de la configuración de su hardware y software y se encripta de una manera en la que no contiene información personal identificable, pero puede usarse para rastrear sus inicios de sesión a nuestro servicio.

Reviewer: ```suggestion
Cuando se conecta a nuestro servicio desde el cliente de juego de osu!, una cadena específica del cliente es subida para ayudarnos a identificar su entorno de juego actual. Se crea a partir de una combinación de identificadores de la configuración de su hardware y software, y se encripta de una manera en la que no contiene información personal identificable, pero puede usarse para rastrear sus inicios de sesión a nuestro servicio.
```

> El principal objetivo de esto es mantener un sistema de clasificación justo y ayudarnos a imponer seguridad en su cuenta en el caso de que la misma sea accedida desde una ubicación comprometida. Esto es considerado privado y solo se almacena durante el tiempo que se considere relevante. Esto también es no transferible, y no tiene sentido fuera del ecosistema de osu!.

Reviewer: ```suggestion
El principal objetivo de esto es mantener un sistema de clasificación justo y ayudarnos a imponer seguridad en su cuenta en el caso de que la misma sea accedida desde una ubicación comprometida. Esto es considerado privado y solo se almacena durante el tiempo que se considere relevante. Esto también es intransferible, y no tiene significado fuera del ecosistema de osu!.
```

> El principal objetivo de esto es mantener un sistema de clasificación justo y ayudarnos a imponer seguridad en su cuenta en el caso de que la misma sea accedida desde una ubicación comprometida. Esto es considerado privado y solo se almacena durante el tiempo que se considera relevante. Esto también es no transferible, y no tiene sentido fuera del ecosistema de osu!

Author: ```suggestion
El principal objetivo de esto es mantener un sistema de clasificación justo y ayudarnos a imponer seguridad en su cuenta en el caso de que la misma sea accedida desde una ubicación comprometida. Esto es considerado privado y solo se almacena durante el tiempo que se considera relevante. Esto también es no transferible, y no tiene sentido fuera del ecosistema de osu!.
```

### Al jugar al juego y subir una puntuación

> Al completar una sesión de juego (completar o fallar un mapa), detalles de su rendimiento se subirán automaticamente a nuestro servidor. La porción de puntuación de este envío incluye datos de la repetición del juego y pueden ser enseñados públicamente en las Clasificaciones Globales y en su Perfil de Usuario y no pueden ser borradas o modificadas.

Author: ```suggestion
Al completar una sesión de juego (completar o fallar un mapa), los detalles de su rendimiento se enviarán automaticamente a nuestro servidor. Una parte de la puntuación que se envía incluyen los datos de la repetición del juego y podrán mostrarse públicamente en las clasificaciones globales y en su perfil de usuario, y no podrán eliminarse ni modificarse.
```

> ### Anti-cheat

Reviewer 2: ```suggestion
### Antitrampas
```

> osu! contiene código ejecutable que es usado para detectar el uso de software de trampas. El objetivo del anti-cheat de osu! es mantener un ambiente justo y competitivo de juego para todos los jugadores, sin afectar al rendimiento del juego o a la privacidad del usuario.

Reviewer 2: ```suggestion
osu! contiene código ejecutable que es usado para detectar el uso de software de trampas. El objetivo del antitrampas de osu! es mantener un ambiente justo y competitivo de juego para todos los jugadores, sin afectar al rendimiento del juego o a la privacidad del usuario.
```

- La detección de trampas ocurre en su máquina y no se envían datos innecesarios a nuestro servidor durante el proceso de detección.

> - Si el antitrampas detecta que estás haciendo trampa, entonces este hecho, además de las pruebas en forma de diagnósticos y metadatos, se envían a nuestros servidores para su verificación. Si no estás haciendo trampa, no se transmitirá ningún dato del antitrampas.

Reviewer 2: ```suggestion
- Si el antitrampas detecta que está haciendo trampa, entonces este hecho, además de las pruebas en forma de diagnósticos y metadatos, se envían a nuestros servidores para su verificación. Si no está haciendo trampa, no se transmitirá ningún dato del antitrampas.
```

> - Si el anti-cheat detecta que está haciendo trampa, entonces este hecho además de las pruebas en forma de diagnósticos y metadatos se envían a nuestros servidores para su verificación. Si no está haciendo trampa, no se enviarán datos del anti-cheat.```

Reviewer 2: ```suggestion
- Si el antitrampas detecta que estás haciendo trampa, entonces este hecho, además de las pruebas en forma de diagnósticos y metadatos, se envían a nuestros servidores para su verificación. Si no estás haciendo trampa, no se transmitirá ningún dato del antitrampas.
```

> - Si el anti-cheat detecta que está haciendo trampas, entonces este hecho en adición a pruebas en forma de diagnósticos y metadatos son enviados a nuestros servidores para la verificación. Si no está haciendo trampas, no se enviarán datos del anti-cheat.

Author: ```suggestion
- Si el anti-cheat detecta que está haciendo trampa, entonces este hecho además de las pruebas en forma de diagnósticos y metadatos se envían a nuestros servidores para su verificación. Si no está haciendo trampa, no se enviarán datos del anti-cheat.```

> - Incluso en la detección, intentamos con nuestro mejor empeño no enviar nunca información fuera del ecosistema de osu! con la cual se le podría identificar personalmente.

Reviewer 2: ```suggestion
- Incluso en caso de detección, se hace todo lo posible por no enviar nunca información fuera del ecosistema de osu! que pueda usarse para identificarlo personalmente.
```

- Los metadatos transmitidos se retienen en nuestro servidor únicamente durante el tiempo que son útiles. Esto es generalmente desde horas hasta días mientras analizamos el contenido reportado.
- El análisis es mayormente automatizado. Los metadatos transmitidos no son accesibles por miembros del equipo de soporte, y deben ser accedidos mediante varias capas de seguridad. Solo los administradores de la base de datos pueden ver los metadatos.

> Respetamos y valoramos su privacidad y no deseamos imponer miedo en usuarios legítimos por tener un anti-cheat presente en el juego.

Author: ```suggestion
Respetamos y valoramos su privacidad y no deseamos imponer miedo en los usuarios legítimos por tener un anti-cheat presente en el juego.
```

> Respetamos y valoramos su privacidad y no deseamos imponer miedo en los usuarios legítimos por tener un anti-cheat presente en el juego.

Reviewer 2: ```suggestion
Respetamos y valoramos su privacidad y no deseamos imponer miedo en los usuarios legítimos por tener un antitrampas presente en el juego.
```

### Registro de eventos

> Utilizamos registro de errores el cual recolecta información técnica y de uso mientras usas nuestros servicios. Esto puede incluir su dirección IP, su nombre de usuario, el tipo de navegador y la versión, el huso horario y ubicación, sistema operativo y plataforma y otros datos sobre que dispositivos usa para acceder a nuestros servicios.

Author: ```suggestion
Usamos el registro de errores, el cual recolecta información técnica y de uso mientras usas nuestros servicios. Esto puede incluir su dirección IP, su nombre de usuario, el tipo de navegador y la versión, el huso horario y ubicación, sistema operativo y plataforma y otros datos sobre que dispositivos usa para acceder a nuestros servicios.
```

> Usamos el registro de errores, el cual recolecta información técnica y de uso mientras usas nuestros servicios. Esto puede incluir su dirección IP, su nombre de usuario, el tipo de navegador y la versión, el huso horario y ubicación, sistema operativo y plataforma y otros datos sobre que dispositivos usa para acceder a nuestros servicios.

Reviewer 2: ```suggestion
Usamos un sistema de registro de errores que recopila información técnica y de uso cuando usa nuestros servicios. Esto puede incluir su dirección IP, su nombre de usuario, el tipo y la versión del navegador, la zona horaria y la ubicación, el sistema operativo y la plataforma, así como otros detalles sobre los dispositivos que usa para acceder a nuestros servicios.
```

> Estos datos recolectados son agregados y solo son retenidos mientras sean útiles. Generalmente, el periodo de retención de datos no agregados es menos de un mes, con reglas de purga automáticas.

Reviewer 2: ```suggestion
Estos datos recopilados se agrupan y solo se conservan mientras sean útiles. Por lo general, el periodo de conservación de los datos no agrupados es inferior a un mes, con reglas de depuración automática.
```

## Divulgación de sus datos personales

> No hacemos nada de marketing, publicidad ni enviamos ningún correo no solicitado. Los únicos correos que recibirá de nosotros serán el resultado de una acción en nuestro servicio (como la solicitud de verificación en dos pasos, comprar un producto o la activación de las notificaciones para una discusión).

Reviewer: ```suggestion
No hacemos nada de mercadotecnia, publicidad ni enviamos ningún correo no solicitado. Los únicos correos que recibirá de nosotros serán el resultado de una acción en nuestro servicio (como la solicitud de verificación en dos pasos, comprar un producto o la activación de las notificaciones para una discusión).
```

> No hacemos nada de marketing, publicidad ni enviamos ningún correo no solicitado. Los únicos correos que recibirás de nosotros serán el resultado de una acción en nuestro servicio (como la solicitud de verificación en dos pasos, comprar un producto o la activación de las notificaciones para una discusión).

Reviewer 2: ```suggestion
No hacemos nada de marketing, publicidad ni enviamos ningún correo no solicitado. Los únicos correos que recibirá de nosotros serán el resultado de una acción en nuestro servicio (como la solicitud de verificación en dos pasos, comprar un producto o la activación de las notificaciones para una discusión).
```

Podremos compartir su información personal con terceros en casos muy específicos:

> - Donde ha hecho información pública expresamente

Author: ```suggestion
- Cuando haya hecho pública expresamente la información
```

> - Para rellenar sus pedidos de la tienda

Author: ```suggestion
- Para tramitar sus pedidos de la tienda
```

- Para procesar pagos mediante proveedores de procesamiento de pagos como Paypal y Xsolla.

> - Para procesar sus tickets de servicio al cliente (usamos [Enchant](https://enchant.com))

Reviewer: ```suggestion
- Para procesar sus tiquetes de servicio al cliente (usamos [Enchant](https://enchant.com))
```

- Para mejorar nuestros servicios, mediante el registro de errores (usamos [Sentry](https://sentry.io))

## Sus derechos y control

> Como usuario, tiene el derecho de migrar, actualizar o eliminar sus datos personales. Esto se puede hacer, desde la [configuración de su cuenta](https://osu.ppy.sh/home/account/edit), o, si no está disponible, desde una función «Editar» localizada en la sección relevante de nuestro sitio web. En caso de que desee recuperar mediante programación los datos completos de su cuenta, use nuestra [API pública](https://github.com/ppy/osu-api/wiki).

Reviewer: ```suggestion
Como usuario, tiene el derecho de migrar, actualizar o eliminar sus datos personales. Esto se puede hacer, desde la [configuración de su cuenta](https://osu.ppy.sh/home/account/edit), o, si no está disponible, desde una función «Editar» localizada en la sección relevante de nuestro sitio web. En caso de que desee recuperar programáticamente los datos completos de su cuenta, use nuestra [API pública](https://github.com/ppy/osu-api/wiki).
```

> Como usuario tiene el derecho de migrar, actualizar o eliminar tus datos personales. Esto se puede hacer, principalmente, desde la [configuración de usuario](https://osu.ppy.sh/home/account/edit), o donde no esté disponible desde una función «Editar» localizada en la sección relevante de nuestro sitio web. En casos en los que desee recuperar programáticamente toda los datos de su cuenta, por favor use nuestra [API pública](https://github.com/ppy/osu-api/wiki).

Author: ```suggestion
Como usuario, tiene el derecho de migrar, actualizar o eliminar tus datos personales. Esto se puede hacer, principalmente, desde la [configuración de tu cuenta](https://osu.ppy.sh/home/account/edit), o, si no está disponible, desde una función «Editar» localizada en la sección relevante de nuestro sitio web. En caso de que desee recuperar mediante programación los datos completos de su cuenta, use nuestra [API pública](https://github.com/ppy/osu-api/wiki).
```

> Como usuario, tiene el derecho de migrar, actualizar o eliminar tus datos personales. Esto se puede hacer, principalmente, desde la [configuración de tu cuenta](https://osu.ppy.sh/home/account/edit), o, si no está disponible, desde una función «Editar» localizada en la sección relevante de nuestro sitio web. En caso de que desee recuperar mediante programación los datos completos de su cuenta, use nuestra [API pública](https://github.com/ppy/osu-api/wiki).

Reviewer 2: ```suggestion
Como usuario, tiene el derecho de migrar, actualizar o eliminar sus datos personales. Esto se puede hacer, desde la [configuración de su cuenta](https://osu.ppy.sh/home/account/edit), o, si no está disponible, desde una función «Editar» localizada en la sección relevante de nuestro sitio web. En caso de que desee recuperar mediante programación los datos completos de su cuenta, use nuestra [API pública](https://github.com/ppy/osu-api/wiki).
```

> En muchos casos, contenido creado por los usuarios como publicaciones del foro y beatmaps pueden ser borrados de forma individual. Encontrará botones de eliminar asociados directamente con los elementos que pueden ser borrados.

Author: ```suggestion
En muchos casos, el contenido creado por los usuarios, como publicaciones del foro y beatmaps, pueden eliminarse de forma individual. Encontrará botones de eliminación asociados directamente con los elementos que pueden ser eliminados.
```

> En muchos casos, el contenido creado por los usuarios, como publicaciones del foro y beatmaps, pueden eliminarse de forma individual. Encontrará botones de eliminación asociados directamente con los elementos que pueden ser eliminados.

Reviewer 2: ```suggestion
En muchos casos, el contenido creado por los usuarios, como publicaciones del foro y beatmaps, pueden eliminarse de forma individual. Encontrará botones de eliminación asociados directamente con los elementos que puedan ser eliminados.
```

> Tienes la opción de eliminar su cuenta de nuestro servicio. Por favor tenga en cuenta que esto actualmente es un proceso manual y puede tomar varios dias para completarse ([contáctanos](mailto:privacy@ppy.sh) para rellenar una solicitud). En el caso del borrado de su cuenta, porciones de sus contribuiciones públicas pueden mantenerse intactas, como está detallado en «Información que recogemos».

Author: ```suggestion
Tiene la opción de eliminar tu cuenta de nuestro servicio. Tenga en cuenta que esto actualmente es un proceso manual y puede tomar varios días para completarse ([contáctenos](mailto:privacy@ppy.sh) para rellenar una solicitud). En el caso de que elimine su cuenta, es posible que parte de sus contribuciones públicas permanezcan intactas, como está detallado en «Información que recogemos».
```

> Tiene la opción de eliminar tu cuenta de nuestro servicio. Tenga en cuenta que esto actualmente es un proceso manual y puede tomar varios días para completarse ([contáctenos](mailto:privacy@ppy.sh) para rellenar una solicitud). En el caso de que elimine su cuenta, es posible que parte de sus contribuciones públicas permanezcan intactas, como está detallado en «Información que recogemos».

Reviewer 2: ```suggestion
Tiene la opción de eliminar su cuenta de nuestro servicio. Tenga en cuenta que esto actualmente es un proceso manual y puede tomar varios días para completarse ([contáctenos](mailto:privacy@ppy.sh) para rellenar una solicitud). En el caso de que elimine su cuenta, es posible que parte de sus contribuciones públicas permanezcan intactas, como está detallado en «Información que recogemos».
```

> Debido a que tenemos una política estricta de una-cuenta-por-usuario para mantener las tablas de clasificación justas, tras eliminar su cuenta no podrá volver a nuestro servicio. Para hacer cumplir esto, también mantendremos la cadena de identificación única mencionada en «Información que recogemos», incluso después de la eliminación de la cuenta. Esto se hace como un [interés legítimo, tal y como se indica en el GDPR](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/legitimate-interests/when-can-we-rely-on-legitimate-interests/), con el fin de hacer cumplir nuestras reglas. Puede estar seguro de que esta información es unidireccional y no se puede usar para identificarte fuera de osu! o más allá del contexto preciso de la aplicación de esta regla.

Reviewer 2: ```suggestion
Debido a que tenemos una política estricta de una cuenta por usuario para mantener las tablas de clasificación justas, es posible que después de la eliminación de su cuenta no pueda volver a nuestro servicio. Para hacer cumplir esto, también mantendremos la cadena de identificación única mencionada en «Información que recogemos», incluso después de la eliminación de la cuenta. Esto se hace como un [interés legítimo, tal y como se indica en el GDPR](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/legitimate-interests/when-can-we-rely-on-legitimate-interests/), con el fin de hacer cumplir nuestras reglas. Puede estar seguro de que esta información es unidireccional y no se puede usar para identificarlo fuera de osu! o más allá del contexto preciso de la aplicación de esta regla.
```

Author: ```suggestion
Debido a que tenemos una política estricta de una-cuenta-por-usuario para mantener las tablas de clasificación justas, es posible que después de la eliminación de su cuenta no pueda volver a nuestro servicio. Para hacer cumplir esto, también mantendremos la cadena de identificación única mencionada en «Información que recogemos», incluso después de la eliminación de la cuenta. Esto se hace como un [interés legítimo, tal y como se indica en el GDPR](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/legitimate-interests/when-can-we-rely-on-legitimate-interests/), con el fin de hacer cumplir nuestras reglas. Puede estar seguro de que esta información es unidireccional y no se puede usar para identificarlo fuera de osu! o más allá del contexto preciso de la aplicación de esta regla.
```

> Debido a que tenemos una política estricta de una-cuenta-por-usuario para mantener las tablas de clasificación justas, tras eliminar su cuenta no podrás volver a nuestro servicio. Para imponer esto, mantendremos también la cadena única identificativa como fue mencionada en «Información que recogemos», incluso después de eliminar la cuenta. Esto se hace de manera que sea [interés legítimo tal y como está recogido en el GDPR](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/legitimate-interests/when-can-we-rely-on-legitimate-interests/) para poder imponer nuestras reglas. Puede estar tranquilo, pues esta información esta encriptada de manera que no se pueda desencriptar y por tanto no se puede usar para identificarte fuera de osu!, o fuera del preciso contexto de imponer esta regla.

Author: ```suggestion
Debido a que tenemos una política estricta de una-cuenta-por-usuario para mantener las tablas de clasificación justas, tras eliminar su cuenta no podrá volver a nuestro servicio. Para hacer cumplir esto, también mantendremos la cadena de identificación única mencionada en «Información que recogemos», incluso después de la eliminación de la cuenta. Esto se hace como un [interés legítimo, tal y como se indica en el GDPR]https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/legitimate-interests/when-can-we-rely-on-legitimate-interests/), con el fin de hacer cumplir nuestras reglas. Puede estar seguro de que esta información es unidireccional y no se puede usar para identificarte fuera de osu! o más allá del contexto preciso de la aplicación de esta regla.
```

> Debido a que tenemos una política estricta de una-cuenta-por-usuario para mantener las tablas de clasificación justas, tras eliminar su cuenta no podrá volver a nuestro servicio. Para hacer cumplir esto, también mantendremos la cadena de identificación única mencionada en «Información que recogemos», incluso después de la eliminación de la cuenta. Esto se hace como un [interés legítimo, tal y como se indica en el GDPR]https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/legitimate-interests/when-can-we-rely-on-legitimate-interests/), con el fin de hacer cumplir nuestras reglas. Puede estar seguro de que esta información es unidireccional y no se puede usar para identificarte fuera de osu! o más allá del contexto preciso de la aplicación de esta regla.

Reviewer 2: ```suggestion
Debido a que tenemos una política estricta de una-cuenta-por-usuario para mantener las tablas de clasificación justas, tras eliminar su cuenta no podrá volver a nuestro servicio. Para hacer cumplir esto, también mantendremos la cadena de identificación única mencionada en «Información que recogemos», incluso después de la eliminación de la cuenta. Esto se hace como un [interés legítimo, tal y como se indica en el GDPR](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/legitimate-interests/when-can-we-rely-on-legitimate-interests/), con el fin de hacer cumplir nuestras reglas. Puede estar seguro de que esta información es unidireccional y no se puede usar para identificarte fuera de osu! o más allá del contexto preciso de la aplicación de esta regla.
```

## Cookies

Usamos cookies, tal y como hace el resto de internet. Las usamos estrictamente para mantener el estado de su sesión y sus credenciales de inicio de sesión entre multiples sesiones. Si es alérgico a las cookies, no use nuestros servicios.

## Seguridad de datos

> La seguridad es muy importante para nosotros. osu! sigue los estándares aceptados para proteger su información personal durante el procesamiento, transferencia y almacenamiento. Usamos HSTS para asegurar que todos los sitios en nuestro dominio están encriptados mediante TLS y que mantienen unos altos estándares en seguridad de datos para el acceso a nuestros servidores, restringiendo el acceso a sus datos personales cuando no necesitamos acceder a ellos.

Author: ```suggestion
La seguridad es muy importante para nosotros. osu! sigue los estándares aceptados para proteger su información personal durante el procesamiento, transferencia y almacenamiento. Usamos HSTS para asegurar que todos los sitios en nuestro dominio están encriptados mediante TLS y que mantienen unos altos estándares en seguridad de datos para el acceso a nuestros servidores, restringiendo el acceso a sus datos personales cuando no necesitemos acceder a ellos.
```

También purgamos datos en un proceso continuo y automático para mantener únicamente los suficientes datos históricos como son necesarios para llevar a cabo nuestros intereses legítimos comerciales.

## Niños

Los servicios de osu! no están diseñados para niños por debajo de la edad de 13 años. Si descubrimos que una persona por debajo de dicha edad nos ha proporcionado información personal sin permiso parental, nos comprometemos a eliminar dicha información de nuestros sistemas.

## Controlador de datos

> Hola, soy Dean (usualmente conocido como peppy) y soy el controlador de sus datos. Si tienes cualquier duda o petición sobre los que llevar a cabo sus derechos legales, no dude en contactarme directamente a la dirección descrita debajo.

Reviewer 2: ```suggestion
Hola, soy Dean (comúnmente conocido como peppy) y soy el controlador de tus datos. Si tienes alguna duda sobre la privacidad o quieres ejercer tus derechos legales, no dudes en ponerte en contacto conmigo directamente en la dirección que aparece a continuación.
```

Author: ```suggestion
Hola, soy Dean (comúnmente conocido como peppy) y soy el controlador de sus datos. Si tiene alguna duda sobre la privacidad o quiere ejercer sus derechos legales, no dude en ponerse en contacto conmigo directamente en la dirección que aparece a continuación.
```

> **Correo electrónico**: [privacy@ppy.sh](mailto:privacy@ppy.sh) (Respuesta en 24 horas garantizada)

Reviewer 2: ```suggestion
**Correo electrónico**: [privacy@ppy.sh](mailto:privacy@ppy.sh)
```
